### PR TITLE
Fix stats scss to use variables.

### PR DESCRIPTION
**Pull request by richbachman**
Updated the stats SCSS file to use the most recent version from style guide.

- [x] Has descriptive PR title and body
- [x] All changes are backwards compatible
- [x] QA'd by the SendGrid Style Guide designer
- [x] Code reviewed by a SendGrid frontend lead

### CODE REVIEW

**Comment by alexabreu** [comment]
lgtm, @richbachman. just want to be sure our storyshots don't need to be updated.

File: src/stats-card/StatsCard.module.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@
   justify-content: space-around;
   width: 100%;
   padding: $scale-base-up-06;
-  border: 1px solid $slate-20;
+  border: 1px solid $slate-10;
```
Comment (alexabreu) [inline src/stats-card/StatsCard.module.scss]: this is the only real change, correct? the other diff looks like formatting? @richbachman

Comment (richbachman) [reply src/stats-card/StatsCard.module.scss]: Yes.

**Comment by richbachman** [comment]
Merging into master for future deploy.

**Comment by thinkingserious** [comment]
Hello @richbachman,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**